generate_nutrition.py

```diff
@@ -147,15 +147,61 @@ def add_recipe(name, servings, ingredients):
     for amount, unit, ingredient in ingredients:
         add_ingredient(amount, unit, ingredient)
 
+def change_meta_data(recipe_name):
+    today = date.today()
+    name_box = driver.find_element(By.XPATH, '//input[@title="New Recipe"]')
+    name_box.click()
+    name_box.clear()
+    name_box.send_keys(recipe_name)
+    notes_box = driver.find_element(By.XPATH,'//textarea[@class="gwt-TextArea"]')
+    notes_box.click()
+    notes_box.send_keys(f"Added on {today.strftime('%d.%m.%Y')}")
+
+def change_serving_size(serving_size):
+    serving_size_image = driver.find_element(By.XPATH, value="//img[@title='Add Measure']")
+    serving_size_image.click()
+    serving_size_field = driver.find_element(By.XPATH, value='//div[text()="Servings Per Recipe"]/parent::td/parent::tr/parent::tbody/tr[2]/td[3]')
+    serving_size_field.click()
+    serving_size_field = driver.find_element(By.XPATH, value='//div[text()="Servings Per Recipe"]/parent::td/parent::tr/parent::tbody/tr[2]/td[3]/input')
+    serving_size_field.send_keys(serving_size)
+
+def save_export_recipe(save_location,save_name):
+    save_button = driver.find_element(By.XPATH, value="//button[text()='Save Changes']")
+    save_button.click()
+    sleep(1) # Wait for website to save the file
+    menu_button = driver.find_element(By.XPATH, value="//div[@class='GO-RHEKCA3']/img")
+    menu_button.click()
+    export_div = driver.find_element(By.XPATH, value="//div[contains(@class, 'gwt-Label') and text()='Export to CSV File...']")
+    export_div.click()
+    sleep(2) # wait for file to be downloaded
```

Comment:
good for now. If possible we should try to move to   `if download completed` if that's possible. Can you add a TODO?